scripts/events_module/relationship/welcoming_events.py

```diff
@@ -75,24 +76,20 @@ def welcome_cat(clan_cat: Cat, new_cat: Cat) -> None:
         change_relationship_values(
             cats_to=[clan_cat],
             cats_from=[new_cat],
-            romantic_love=new_to_clan_cat["romantic"],
-            platonic_like=new_to_clan_cat["platonic"],
-            dislike=new_to_clan_cat["dislike"],
-            admiration=new_to_clan_cat["admiration"],
-            comfortable=new_to_clan_cat["comfortable"],
-            jealousy=new_to_clan_cat["jealousy"],
-            trust=new_to_clan_cat["trust"],
+            romance=new_to_clan_cat[RelValue.ROMANCE],
+            like=new_to_clan_cat[RelValue.LIKE],
+            respect=new_to_clan_cat[RelValue.RESPECT],
+            comfort=new_to_clan_cat[RelValue.COMFORT],
+            trust=new_to_clan_cat[RelValue.TRUST],
```

Comment:
you already know what i'm gonna say here, but...

```suggestion
            **new_to_clan_cat
```